packages/protocol-contracts/contracts/interfaces/ZetaInterfaces.sol

```diff
@@ -39,7 +39,7 @@ interface ZetaInterfaces {
         uint256 sourceChainId;
         bytes destinationAddress;
         uint256 destinationChainId;
-        uint256 zetaValueAndGas;
+        uint256 remainingZetaValue;
```

Comment:
Can we add this 2 comments?
https://github.com/zeta-chain/zetachain/compare/main...zeta-1104